package.json

```diff
@@ -40,8 +40,10 @@
     "@bower_components/crypto": "anodynos/node2web_crypto#3.46.1",
     "@bower_components/international-phone-number": "mareczek/international-phone-number#^0.0.16",
     "@bower_components/matchmedia": "paulirish/matchMedia.js#~0.2.0",
+    "@ovh-ux/manager-config": "^0.2.0",
```

Comment:
Duplicate from a pending pull request: #1383

Reply:
This is to avoid being dependent from a branch other than `develop` but to be able to test and build, I will rebase once https://github.com/ovh-ux/ovh-manager-telecom/pull/1383 is merged